tests/ga/test_update.py

```diff
@@ -1915,6 +1915,7 @@ def http_get_handler(url, *_, **__):
             update_handler = get_update_handler()
             self.assertFalse(update_handler._try_update_goal_state(protocol), "try_update_goal_state should have failed")
 
+    @skip_if_predicate_true(lambda: True, "TODO: Enable this test once we start retrieving the ExtensionsGoalState (vmSettings)")
```

Comment:
The test will be re-enabled once we actually start using the vmSettings to drive the goal state